tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "noEmit": true,
     "strict": true,
     "esModuleInterop": true,
-    "moduleResolution": "nodenext",
+    "moduleResolution": "NodeNext",
+    "module": "esnext",
```

Comment:
I don't know if this change is safe or compatible with the build system (nsm). Do we need to introduce it? Or safe to revert?

Reply:
Typescript was giving me error to put it there

Reply:
Can you share the error? CI is passing so it must be local to your machine. I recall there is a dependency or build conflict the last time I tried to change this, which it why I am asking.